config/detekt/detekt.yml

```
@@ -268,7 +268,7 @@ exceptions:
       - 'Throwable'
     allowedExceptionNameRegex: '_|(ignore|expected).*'
   TooGenericExceptionThrown:
-    active: true
+    active: false
```

```suggestion
    active: true
```
imo this can stay true. It's only about generic exceptions being caught, since that can act as a fallback. As long as its not swallowed (i.e. not logged) its fine.